tutorials/general/100_getting_started.ipynb

```diff
@@ -557,9 +557,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    ":::{image} ../_static/img/100_cellrank_kernels_mode.png\n",
+    ":class: only-dark\n",
+    ":width: 800px\n",
+    ":::\n",
+    "\n",
+    ":::{figure-md} fig-structure\n",
```

Comment:
Please remove the `:::{figure-md} fig-structure` part - there are now 2 images being rendered.